test/langtools/tools/javac/diags/examples/ExtraImportSemicolonWarning.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+// key: compiler.warn.extraneous.semicolon
+// key: compiler.warn.source.no.system.modules.path
+// options: --source 20
```

Comment:
Please use "--release 20" instead if possible.

Reply:
> Please use "--release 20" instead if possible.

Thanks - that works better.

Reply:
Per [this CSR comment](https://bugs.openjdk.org/browse/JDK-8301905?focusedCommentId=14559000&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-14559000) it was requested to make this an error for source versions >= 21 and a warning for source versions prior to 21.

In order to satisfy `diags/CheckExamples.java` I needed an example that would generate the warning.

But I guess you're right... so the alternative would be to add `compiler.warn.extraneous.semicolon` to `diags/examples.not-yet.txt`?

Reply:
> What happens if you remove the option completely?

Then the `diags/CheckExamples.java` test fails when it runs against `ExtraImportSemicolonWarning.java`, because that source file will then generate an error instead of a warning as expected.